client/app/configuration/configuration.html

```diff
@@ -39,14 +39,21 @@ <h2 class="oui-tile__title" data-ng-bind="'core_type_of_guide_' + key | translat
                                 <li class="oui-tile__item" data-ng-repeat="guide in value track by $index">
                                     <div class="oui-tile__definition">
                                         <a target="_blank"
+                                            data-track-on="click"
```

Comment:
I think you can also add: `rel="noopener"` for both anchors.

Reply:
have added